cmd/netgoal/README.md

```diff
@@ -0,0 +1,52 @@
+# Netgoal
+
+## netgoal generate
+`--participation-host-machines (-N)` and `--npn-host-machines (-X)` are optional parameters and they default to `--participation-algod-nodes (-n)` and `--npn-algod-nodes (-x)` respectively, i.e. defaults to a machine per algod node.
+
+### Long-Form Flags Example
+- Wallets: The command below will generate 100 wallets for the 100 participation algod nodes. By default each npn gets one wallet each. If there are more wallets than nodes, it will get split across the participation algod nodes.
+- Relays: 8 Relays and 8 machines to host the relays will be generated
+- Participation Nodes: 100 particiipation algod nodes will be distributed across 20 host machines.
+- Non-Participation Nodes (NPNs): 10 non-participation algod nodes will be distributed across 5 host machines.
+
+```
+netgoal generate -t net -r /tmp/wat -o net.json --wallets 100 --relays 8 --participation-host-machines 20 --participation-algod-nodes 100 --npn-host-machines 5 --npn-algod-nodes 10 --node-template node.json --relay-template relay.json --non-participating-node-template nonPartNode.json
+```
+
+### Short-Form Flags Example
+The following will result in the same outcome as the command above.
+```
+netgoal generate -t net -r /tmp/wat -o net.json -w 100 -R 8 -N 20 -n 100 -X 5 -x 10 --node-template node.json --relay-template relay.json --non-participating-node-template nonPartNode.json
```

Comment:
hmm, it's still using shorthand flags here in readme. Should we set a better example by using long form flags?

Reply:
The long form flags example is right above the shorthand flags in the readme.

Reply:
Might be easier to read in this format: https://github.com/algorand/go-algorand/blob/af0c57a67fc3cb6a88990f69d56f4565d6f79725/cmd/netgoal/README.md